Core/Core/ActAsUser/ActAsUserViewController.storyboard

```diff
@@ -139,5 +137,8 @@
     <resources>
         <image name="RedPanda" width="75" height="103"/>
         <image name="WhitePanda" width="84" height="103"/>
+        <namedColor name="backgroundLightest">
+            <color red="0.17647058823529413" green="0.23137254901960785" blue="0.27058823529411763" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+        </namedColor>
```

Comment:
storyboard named colors work fully in iOS 13, choosing by contrast & appearance. iOS 12 will always use the standard color. `backgroundLightest` is pretty safe even for iOS 12 though since 12 doesn't have dark mode, and the color is the same in high contrast.